docs/components/rule.md

```diff
@@ -416,6 +416,7 @@ alertmanagers:
   scheme: http
   path_prefix: ""
   timeout: 10s
+  api_version: v1
```

Comment:
Maybe we could add one sentence on what possible values are supported i.e. `v1` and `v2`?

Reply:
added below as I don't know how we can add it directly here.

Reply:
Yes, that's what I mean - thank you 👍